tests/ops/integration_tests/test_mongo_task.py

```diff
@@ -575,20 +575,26 @@ async def test_object_querying_mongo(
     assert len(filtered_results["mongo_test:customer_details"]) == 1
 
     # Array of embedded emergency contacts returned, array of children, nested array of workplace_info.direct_reports
-    assert filtered_results["mongo_test:customer_details"][0] == {
-        "birthday": datetime(1988, 1, 10, 0, 0),
-        "gender": "male",
-        "customer_id": 1.0,
-        "children": ["Christopher Customer", "Courtney Customer"],
-        "emergency_contacts": [
-            {"name": "June Customer", "phone": "444-444-4444"},
-            {"name": "Josh Customer", "phone": "111-111-111"},
-        ],
-        "workplace_info": {
-            "position": "Chief Strategist",
-            "direct_reports": ["Robbie Margo", "Sully Hunter"],
-        },
+    assert filtered_results["mongo_test:customer_details"][0]["birthday"] == datetime(
+        1988, 1, 10, 0, 0
+    )
+    assert filtered_results["mongo_test:customer_details"][0]["gender"] == "male"
+    assert filtered_results["mongo_test:customer_details"][0]["customer_id"] == 1.0
+    assert filtered_results["mongo_test:customer_details"][0]["children"] == [
+        "Christopher Customer",
+        "Courtney Customer",
+    ]
+    assert filtered_results["mongo_test:customer_details"][0]["emergency_contacts"] == [
+        {"name": "June Customer", "phone": "444-444-4444"},
+        {"name": "Josh Customer", "phone": "111-111-111"},
+    ]
+    assert filtered_results["mongo_test:customer_details"][0]["workplace_info"] == {
+        "position": "Chief Strategist",
+        "direct_reports": ["Robbie Margo", "Sully Hunter"],
     }
+    assert isinstance(
+        filtered_results["mongo_test:customer_details"][0]["_id"], ObjectId
+    )
```

Comment:
The `ObjectId`s are non-deterministic, so instead of testing for the value I've tested on the type